docs/index.md

```diff
@@ -13,13 +13,13 @@ for hosting your (binary) artifacts, regardless or your repository layout.
 
 ??? success "Cron Tasks"
     
-    We have various cron tasks available to automate almost anything like rebuilding metadata, emptying Trash, etc.
+    We have various cron tasks available to automate almost anything like rebuilding metadata, emptying trash, etc.
     
     * [x] Java implementation
     * [x] Groovy implementation
 
 ??? success "Event handling"
-
+(empty section with no information but still have a drop down)
```

Comment:
@steve-todorov : Anything we need to add/correct here?

Reply:
Nothing needs to be corrected. Just wanted to bring to attention that it was empty in case the adding of the information got overlooked.

Reply:
In this case, could you please remove the comment you added in the code? Thanks!

Reply:
You can check out the live preview of your [PR](https://deploy-preview-29--strongbox.netlify.com/). 
If this is going to be added, then you will need to add extra white space in the beginning for it to be displayed properly.

Reply:
So I am unsure how to edit an existing pull request to delete the code.

Reply:
I updated and committed the changes I believe. Please verify that this was correct.